pom.xml

```diff
@@ -7,7 +7,7 @@
     <name>EqualsVerifier</name>
     <version>3.8.2-SNAPSHOT</version>
     <description>EqualsVerifier can be used in JUnit unit tests to verify whether the contract for the equals and hashCode methods is met.</description>
-    <url>http://www.jqno.nl/equalsverifier</url>
+    <url>https:///www.jqno.nl/equalsverifier</url>
```

Comment:
Too many slashes?

Reply:
good catch!

Reply:
Indeed, good catch! Since this PR is already merged, I'll fix it on `main` directly.